doc/DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/emline-SPECTROGRAPH-TILEID-GROUPID.rst

```diff
@@ -53,113 +47,127 @@ Required Header Keywords
-HBETA_SIGMA_IVAR  float32       label for field  48
-HBETA_CONT        float32       label for field  49
-HBETA_CONT_IVAR   float32       label for field  50
-HBETA_SHARE       float32       label for field  51
-HBETA_SHARE_IVAR  float32       label for field  52
-HBETA_EW          float32       label for field  53
-HBETA_EW_IVAR     float32       label for field  54
-HBETA_CHI2        float32       label for field  55
-HBETA_NDOF        int32         label for field  56
-OIII_FLUX         float32       label for field  57
-OIII_FLUX_IVAR    float32       label for field  58
-OIII_SIGMA        float32       label for field  59
-OIII_SIGMA_IVAR   float32       label for field  60
-OIII_CONT         float32       label for field  61
-OIII_CONT_IVAR    float32       label for field  62
-OIII_SHARE        float32       label for field  63
-OIII_SHARE_IVAR   float32       label for field  64
-OIII_EW           float32       label for field  65
-OIII_EW_IVAR      float32       label for field  66
-OIII_CHI2         float32       label for field  67
-OIII_NDOF         int32         label for field  68
-HALPHA_FLUX       float32       label for field  69
-HALPHA_FLUX_IVAR  float32       label for field  70
-HALPHA_SIGMA      float32       label for field  71
-HALPHA_SIGMA_IVAR float32       label for field  72
-HALPHA_CONT       float32       label for field  73
-HALPHA_CONT_IVAR  float32       label for field  74
-HALPHA_SHARE      float32       label for field  75
-HALPHA_SHARE_IVAR float32       label for field  76
-HALPHA_EW         float32       label for field  77
-HALPHA_EW_IVAR    float32       label for field  78
-HALPHA_CHI2       float32       label for field  79
-HALPHA_NDOF       int32         label for field  80
-================= ======= ===== ===================
+================= ======= =============================== ===================
+Name              Type    Units                           Description
+================= ======= =============================== ===================
+TARGETID          int64                                   Unique targeting ID
+Z                 float64                                 Redshift used for fitting (from the redrock file)
+ZWARN             int64                                   Redshift warning flag (from the redrock file)
+SPECTYPE          char[6]                                 Spectroscopic type (from the redrock file)
+DELTACHI2         float64                                 Chi2 difference between the first- and second best redshifts (from the redrock file)
+TARGET_RA         float64 deg                             Right ascension (from the redrock file)
+TARGET_DEC        float64 deg                             Declination (from the redrock file)
+OBJTYPE           char[3]                                 TGT, SKY, BAD, empty (from the redrock file)
+OII_FLUX          float32 1e-17 erg/s/cm^2                Fitted flux for the OII doublet
```

Comment:
`10**-17 erg/(s cm2 Angstrom)` is the standard unit.  Only one `/` is allowed.  For the inverse variance: ``10**+34 (s2 cm4 Angstrom2) / erg2``